load-balancer/api-docs/f5-api-reference/f5-load-balancer.rst

```diff
@@ -365,6 +386,144 @@ Returns statistics for the specified node.
         ]
     }
 
+Disable Node for Maintenance
+----------------------------
+
+This setting allows the Node (all services on an IP address) to accept only new connections that match an existing persistence session.  
+Use this feature to prevent new connections to a Node without affecting existing client connections on the same Node.
+
+To re-enable the Node, see: `Enable Node After Maintenance`_.
+
+*Note: It is important to understand differences between* `Disable vs Offline`_.
+
+::
+
+    PUT /nodes/{node_ID}
+
+Request body
+^^^^^^^^^^^^
+
+::
+
+    {
+        "session": "user-disabled"
+    }
+
+Response
+^^^^^^^^
+
+::
+
```

Comment:
https://pages.github.rackspace.com/automationIntegration/bpi.api.python.v2/f5-loadbalancer.html#nodes__nodeid__put 

internal raml docs shows request payload as 
{
	"appService": null,
	"connectionLimit": 2,
	"description": "Updated node",
	"dynamicRatio": 11,
	"logging": "enabled",
	"rateLimit": "disabled",
	"ratio": 1
}

Reply:
was able to find it in latest updated internal raml docs. Ignore above comment